enricher/fabric8/src/main/java/io/fabric8/maven/enricher/fabric8/WildFlySwarmHealthCheckEnricher.java

```diff
@@ -0,0 +1,80 @@
+package io.fabric8.maven.enricher.fabric8;
+
+import io.fabric8.kubernetes.api.model.Probe;
+import io.fabric8.kubernetes.api.model.ProbeBuilder;
+import io.fabric8.maven.core.util.Configs;
+import io.fabric8.maven.enricher.api.AbstractHealthCheckEnricher;
+import io.fabric8.maven.enricher.api.EnricherContext;
+
+import static io.fabric8.maven.core.util.MavenUtil.hasDependency;
+
+/**
+ * Enriches wildfly-swarm containers with health checks if the monitoring fraction is present.
+ */
+public class WildFlySwarmHealthCheckEnricher extends AbstractHealthCheckEnricher {
+
+    public WildFlySwarmHealthCheckEnricher(EnricherContext buildContext) {
+        super(buildContext, "wildfly-swarm-health-check");
+    }
+
+    // Available configuration keys
+    private enum Config implements Configs.Key {
+
+        defaultScheme {{
```

Comment:
As these are actually the names of the properties used, they shouldn't have the `default` prefix AFAIK. @rhuss - halp?